module_utils/helpers.py

```diff
@@ -437,6 +444,21 @@ def get_control_console(socket_path):
     return Console(socket_path)
 
 
+def get_control_console_if_started(console_sock):
+    if not os.path.exists(console_sock):
+        # Console socket doesn't exists
+        return None, None
+
+    try:
+        return get_control_console(console_sock), None
+    except CartridgeException as e:
+        if e.is_instance_not_started_error():
+            # Impossible to connect to socket
```

Comment:
All except branch is "Impossible to connect". BTW seems that it should be an error for `strict` mode.